lib/presentation/destinations/weather/home/widgets/home_page_body/home_page_body_content.dart

```diff
@@ -1,47 +1,32 @@
 import 'package:flutter/material.dart';
-import 'package:flutter_template/foundation/extensions/context_ext.dart';
 import 'package:flutter_template/presentation/base/intent/intent_handler_callback.dart';
-import 'package:flutter_template/presentation/base/widgets/list/ui_list.dart';
 import 'package:flutter_template/presentation/base/widgets/responsive/responsive_builder.dart';
 import 'package:flutter_template/presentation/destinations/weather/home/home_screen_intent.dart';
-import 'package:flutter_template/presentation/entity/base/ui_list_item.dart';
-import 'package:flutter_template/presentation/entity/weather/ui_weather.dart';
-import 'package:flutter_template/presentation/intl/translations/translation_keys.dart';
-
-import '../list/ui_weather_renderer.dart';
 
 class HomePageBodyContent extends StatelessWidget {
-  final List<UIListItem> weatherList;
+  final List<Widget> svgThumbnails; // Changed from UIListItem to Widget
```

Comment:
As the svgThumbnails are in a global state with riverpod, you can directly get the data from it.
No need to import from the parent widget.

Reply:
ok

Reply:
ok